docs/integrations/sources/google-analytics-v4.md

```diff
@@ -190,6 +190,7 @@ Incremental sync is supported only if you add `ga:date` dimension to your custom
 
 | Version | Date       | Pull Request                                             | Subject                                                                                      |
 | :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------- |
+| 0.2.1   | 2023-07-11 | [28149](https://github.com/airbytehq/airbyte/pull/28149) | Specify date format to support datepicker in UI                                                                         |
```

Comment:
Please delete extra space after `Specify date format to support datepicker in UI`.

Reply:
removed space here